nntrainer/tensor/tensor.cpp

```diff
@@ -738,6 +740,14 @@ Tensor Tensor::sum(const std::vector<unsigned int> &axes, float alpha) const {
   return sum(axes, ret, alpha);
 }
 
+void Tensor::merge_axis(unsigned int axis1, unsigned int axis2) {
+  if (axis2 != axis1 + 1)
+    throw std::invalid_argument("Axis to be merged must be continuous.");
```

Comment:
is it also considered continuous if axis1 > axis2 ?

Reply:
Update the error message to be more precise. We only merge is axis2 == axis1 + 1.

Reply:
Cool